docs/introduction/domain-settings.md

```diff
@@ -0,0 +1,147 @@
+# Domain Settings
+
+Při práci se Shopsys Frameworkem je možné se setkat s pojmy - doména, multidoménovost, multijazyčnost.
+V tomto dokumentu se objasníme co tyto pojmy znamenají a jak s nimi pracovat při vývoji vašeho projektu.
+
+## Doména, multidoménovost, multijazyčnost
+
+- **Doména** - Doménu je možné chápat jako jednu instanci dat eshopu.
+Například na doméně shopsys-furniture.com se muže zobrazovat nábytek a na doméně shopsys-electro.com se muže zobrazovat elektronika.
+Pořád se ale jedná o jednu aplikaci s jedním katalogem.
+Na jednotlivé domény přistupujeme prostřednictvím individuálních url adres.
+Všechny domény mají jednu společnou administraci.
+
+- **Multidoménový atribut** - Hodnota tohoto atributu muže být nastavena ruzně pro každou doménu.
+Příkladem multidoménového atributu je výchozí cenová skupina pro nepřihlášeného zákazníka, tato muže být ruzná pro každou doménu.
+Příkladem nemultidoménového atributu je ean produktu, tento je stejný pro všechny domény. 
+
+- **Multijazyčný atribut** - Hodnota tohoto atributu muže být nastavena ruzně pro každý jazyk.
+Příkladem multijazyčného atributu je název produktu.
+Příkladem nemultijazyčného atributu je cena produktu. 
+
+- **Rozdíl mezi multidoménovým a multijazyčným atributem** - Hodnota multijazyčného atributu bude stejná pro každou doménu se stejným jazykem.
+Např v případě, že je pro jazyk *en* nastaven název produktu *A4tech mouse*, bude tento název stejný pro všechny domény, které mají nastaven jazyk *en*.
+V případě multidoménového atributu je možné nastavovat ruzné hodnoty pro ruzné domény bez ohledu na jazyk
+
+*Pozn: Demodata, které jsou na Shopsys Frameworku k dispozici obsahují data jen v cz a en jazyce*
+
+## Nastavovaní a práce s doménami
+
+### 1. Vytvoření aplikace s jednou doménou
+
+#### 1.1 Konfigurace domény
+Upravte konfiguraci domény v `app/config/domains.yml`.
+V tomto souboru se nastavuje ID domény, identifikátor domény pro doménové záložky v administraci, jazyk domény.
+
+#### 1.2 Konfigurace url adresy
+Nastavte požadovanou url adresu domény v `app/config/domains_urls.yml`.
+
+#### 1.3 Nastavevení projektu jako jednodoménový
+Upravte nastavení parametru `is-multidomain` v `build.xml` na hodnotu `false`.
+Prostřednictvím této změny se upraví chování některých phingových targetu - např při importu demo dat se aplikace neppokusí importovat zbytečně data pro další doménu.
+
+#### 1.4 Nastavení jazyka
+Nastavení jazyka domény proveďte podle instrukcí ze sekce [Nastavení jazyka](#3-nastaveni-jazyka)
```

Comment:
The link does not work properly for some reason (try using "./domain-settings.md#3-nastaveni-jazyka", it should work)